### ISSUE TITLE
resize-to-contents weirdness

### ISSUE THREAD

**Issue by jleonard-r7** [oliyh/kamera#34]
on mac (chrome), it does resize to _near_ contents (in my case, the bottom edge is barely cutoff).
but on linux (chrome), it is cutoff by at least 10% (on the same edge).

**Comment by oliyh** [comment]
Hi,

I have been using it on Linux and I think it seems to work fine. I guess this is related to #29 ? What version of Chrome are you using?

**Comment by jleonard-r7** [comment]
No, it’s not relayed to #29 as our code didn’t change between the two platforms. It’s just a situation where we take the codes we normally run on Mac and then run them on Linux and the screenshots are not resized larger (in either direction) to fit the contents.
